pyproject.toml

```diff
@@ -26,10 +26,10 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 cython = "^0.29.14"
-google-cloud-storage = ">=1.24.1 <=1.28.1"
+google-cloud-storage = ">=1.31.1"
 jupyter = "^1.0.0"
 kornia = "^0.1.4"
-numpy = ">=1.17 < 1.18"
+numpy = ">=1.18.2"
```

Comment:
Let's use `^1.18.2` or `>=1.18.2 <2.0.0` (they are equivalent syntex). This prevents for unexpected major version bump.

Reply:
ah, good call! making that update.